flink-python/pyflink/datastream/tests/test_data_stream.py

```diff
@@ -149,6 +149,22 @@ def flat_map(value):
         expected.sort()
         self.assertEqual(expected, results)
 
+    def test_add_sink_with_sink_func_class(self):
```

Comment:
The test is useless here, as it uses the DataStreamCollectUtil to collect and verify. The `add_sink` has not been tested. You can add a CustomSinkFunction which extends SinkFunction. The code looks like:
```
class TestSinkFunction(SinkFunction):
   def __init__(self, func):
      ...
   def get_results():
     ...
```